### PR TITLE
[CAS-762] Fix - Remove in distinct channel

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-762](https://stream-io.atlassian.net/browse/CAS-762)
### Description

Removing option to remove user of a distinct channel.

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog updated with client-facing changes
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt [hunk 1]
```diff
@@ -28,6 +28,11 @@ public fun Channel.isDistinctChannel(): Boolean {
     return cid.contains("!members")
 }
 
+@InternalStreamChatApi
+public fun String.isDistinctChannel(): Boolean {
```
Comment (adasiewiczr) [inline stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt:32]: The same as in #1544  - I would rather move this to sample app package

Comment (bychkovdmitry) [inline stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt:32]: A bit of duplication here. I would do in the function above:
```
@InternalStreamChatApi
public fun Channel.isDistinctChannel(): Boolean {
    return cid.isDistinctChannel()
}
```
Also in `GroupChatInfoFragment`
```
private fun isDistinctChannel(): Boolean = args.cid.isDistinctChannel()
```

**Review by bychkovdmitry — APPROVED** [review]

File: stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt (hunk 1 above)
Comment (leandroBorgesFerreira) [reply stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/extensions/Channel.kt]: Handled in a different PR.

File: stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/member/GroupChatInfoMemberOptionsDialogFragment.kt [hunk 2]
```diff
@@ -53,6 +55,7 @@ class GroupChatInfoMemberOptionsDialogFragment : BottomSheetDialogFragment() {
 
     override fun getTheme(): Int = R.style.StreamUiBottomSheetDialogTheme
 
+    @InternalStreamChatApi
```
Comment (adasiewiczr) [inline stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/member/GroupChatInfoMemberOptionsDialogFragment.kt:58]: We can remove that now

**Pull request merged**